### PR TITLE
Create static pages for application

### PR DESCRIPTION
**Pull request by ducminhvu**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,45 +1,19 @@
 source 'https://rubygems.org'
 
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.1'
-# Use sqlite3 as the database for Active Record
-gem 'sqlite3'
-# Use SCSS for stylesheets
+gem 'bootstrap-sass', '~> 3.3.5'
```
Comment (dieunb) [inline Gemfile]: replace all `'` by `"`

File: app/assets/javascripts/static_pages.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (dieunb) [inline app/assets/javascripts/static_pages.coffee]: remove this file

File: app/helpers/static_pages_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (dieunb) [inline app/helpers/static_pages_helper.rb]: remove this file

File: app/views/static_pages/about.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<% provide(:title, t("about")) %>
```
Comment (dieunb) [inline app/views/static_pages/about.html.erb]: remove `()`

File: app/views/static_pages/contact.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<% provide(:title, t("contact")) %>
```
Comment (dieunb) [inline app/views/static_pages/contact.html.erb]: remove `()`

File: app/views/static_pages/help.html.erb [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<% provide(:title, t("help")) %>
```
Comment (dieunb) [inline app/views/static_pages/help.html.erb]: remove `()`

File: app/views/static_pages/home.html.erb [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<% provide(:title, t("home")) %>
```
Comment (dieunb) [inline app/views/static_pages/home.html.erb]: remove `()`

File: config/database.yml.example [hunk 8]
```diff
@@ -1,25 +0,0 @@
-# SQLite version 3.x
```
Comment (dieunb) [inline config/database.yml.example]: don't remove this file. Checkout it

File: config/routes.rb [hunk 9]
```diff
@@ -1,56 +1,8 @@
...
-  # Example resource route within a namespace:
-  #   namespace :admin do
-  #     # Directs /admin/products/* to Admin::ProductsController
-  #     # (app/controllers/admin/products_controller.rb)
-  #     resources :products
-  #   end
+  get "help"      => "static_pages#help"
+  get "about"     => "static_pages#about"
+  get "contact"   => "static_pages#contact"
+  get "signup"    => "users#new"
```
Comment (dieunb) [inline config/routes.rb]: ??? Where?

File: app/views/layouts/application.html.erb [hunk 10]
```diff
@@ -1,14 +1,18 @@
...
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title yield :title %></title>
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: replace `'` with `"`

File: app/views/layouts/_header.html.erb [hunk 11]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("title"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("home"), root_path %></li>
+        <li><%= link_to t("help"), help_path %></li>
+        <li><%= link_to t("log_in"), "#" %></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: Remove ()

File: app/views/layouts/_header.html.erb [hunk 12]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("title"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("home"), root_path %></li>
+        <li><%= link_to t("help"), help_path %></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: Remove ( )

File: app/views/layouts/_header.html.erb [hunk 13]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t("title"), root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to t("home"), root_path %></li>
```
Comment (phamkykhoi) [inline app/views/layouts/_header.html.erb]: Remove ( )

File: app/views/layouts/application.html.erb [hunk 14]
```diff
@@ -1,14 +1,18 @@
...
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title yield :title %></title>
+    <%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
```
Comment (phamkykhoi) [inline app/views/layouts/application.html.erb]: not use =>
use :

File: app/views/layouts/application.html.erb [hunk 15]
```diff
@@ -1,14 +1,18 @@
...
+    <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```
Comment (phamkykhoi) [inline app/views/layouts/application.html.erb]: not use =>
use :

**Comment by phamkykhoi** [comment]
OK

File: config/routes.rb [hunk 16]
```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  root               "static_pages#home"
+
+  get "help"      => "static_pages#help"
+  get "about"     => "static_pages#about"
+  get "contact"   => "static_pages#contact"
+  get "signup"    => "users#new"
```
Comment (dieunb) [inline config/routes.rb]: Remove it

**Comment by dieunb** [comment]
OK

**Pull request merged**